pipelines/base/template-build/template-build.yaml

```diff
@@ -111,6 +111,10 @@ spec:
       value: "$(tasks.build-container.results.IMAGE_URL)"
     - name: IMAGE_DIGEST
       value: "$(tasks.build-container.results.IMAGE_DIGEST)"
+    - name: CHAINS-GIT_URL
```

Comment:
it's generic result not related to CHAINS, so it could be just GIT_URL

Reply:
Yes, but Chains specifically looks for results with these names. There's some [upstream conversation](https://github.com/tektoncd/community/pull/697) in making this better, but that is far off. We need these results to be named this way in the foreseeable future.